<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeperationDialogClass</class>
 <widget class="QDialog" name="SeperationDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="ActorsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ActorListLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Actor List:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FirstActorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>First Actor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SecondActorButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Second Actor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="beginSearch">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Connection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
